inference-engine/tests/ngraph_functions/src/reduce.cpp

```diff
@@ -14,24 +14,26 @@ std::shared_ptr<ngraph::Node> makeReduce(const ngraph::Output<Node>& data,
                                          bool keepDims,
                                          ngraph::helpers::ReductionType reductionType) {
     switch (reductionType) {
-        case helpers::Mean:
-            return std::make_shared<ngraph::opset3::ReduceMean>(data, axes, keepDims);
-        case helpers::Max:
-            return std::make_shared<ngraph::opset3::ReduceMax>(data, axes, keepDims);
-        case helpers::Min:
-            return std::make_shared<ngraph::opset3::ReduceMin>(data, axes, keepDims);
-        case helpers::Prod:
-            return std::make_shared<ngraph::opset3::ReduceProd>(data, axes, keepDims);
-        case helpers::Sum:
-            return std::make_shared<ngraph::opset3::ReduceSum>(data, axes, keepDims);
-        case helpers::LogicalOr:
-            return std::make_shared<ngraph::opset3::LogicalOr>(data, axes);
-        case helpers::LogicalAnd:
-            return std::make_shared<ngraph::opset3::LogicalAnd>(data, axes);
-        case helpers::LogicalXor:
-            return std::make_shared<ngraph::opset3::LogicalXor>(data, axes);
```

Comment:
It looks like LogicalXor was accidentally deleted by this change

Reply:
Because in current version, reduction logical types are incorrectly pointed to `ngraph::opset3::Logical*` which require two tensors that have identical shape and this is not what we want in reduce ops. 
After checking latest version in [upstream](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_helpers/ngraph_functions/src/reduce.cpp#L31), reduction logical types are updated to `ngraph::opset4::ReduceLogical*`, and `ReduceLogicalXor` does not exist, so I removed this type here.